ui-app/client/src/components/ProgressBar/progressBarProperties.ts

```diff
@@ -60,6 +60,97 @@ const propertiesDefinition: Array<ComponentPropertyDefinition> = [
 		group: ComponentPropertyGroup.ADVANCED,
 	},
 	COMMON_COMPONENT_PROPERTIES.visibility,
+
+	{
+		name: 'progressBarDesignSelection',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Progress Bar Selection Type',
+		description: 'Type of the selection of a Progress Bar',
+		defaultValue: '_progressBarDesignOne',
+		group: ComponentPropertyGroup.DATA,
+		enumValues: [
+			{
+				name: '_progressBarDesignOne',
+				displayName: 'Progress Bar Design One',
+				description: 'Progress Bar design one',
+			},
+			{
+				name: '_simpleMenuDesign2',
```

Comment:
Why are we using menu design names in the progress bar??? Please update, this will intersect with the menu component.